breeze

```diff
@@ -117,6 +119,9 @@ function breeze::setup_default_breeze_constants() {
     # If set to true, the database will be initialized, a user created and webserver and scheduler started
     export START_AIRFLOW="false"
 
+    export AIRFLOW__CORE__LOAD_EXAMPLES="true"
```

Comment:
You need to set those variables in the _initialization.sh rather than here. I suggest a new function `initialization::intialize_airflow_variables()`. 

The way it works is, that default values for those variables are set in the _initialization.sh (for both CI and Breeze) and here "default_breeze_constants" sets only the values for those variables that are different than the default. If you specify values in the switches, they will override the default values set in the __initialization. After __initialization is complete most of the variables are set to read-only so that we detect any accidental overriding immediately and avoid any difficult to diagnose mistakes.

Breeze is really the "user-facing" script that uses the "scripts/ci" scripts under-the-hood, but the CI does not use Breeze. 

Context and reasoning:

There are certain variables that should be set by default differently in the development environment and in the CI". In the development environment you mostly already have the environment/images etc. set-up and want to take advantage of it, but in CI you always start from a clean state on a fresh machine and you need to get everything from the scratch as fast as possible and you have the GITHUB_TOKEN, images containing latest sources built in the cache by the "Image Build" workflow, and GCR cache handy. So you can do it much faster and better if you optimize it, and having different default variables helps with achieving all the optimisations.

Reply:
And. side comment - maybe there is some redundancy here. I think there are a few that could be removed from here (historical reasons why they are here). For example DB_RESET could be removed (default in __initialization.sh is also FALSE. Probbaly few others as well.

Reply:
No sure if I follow, I should:
- Remove the variables from breeze 
- Set them in _initialization.sh (in the new function that you suggested)

But it is not clean when the new function should be executed.

Reply:
I have submitted a new commit with the changes

Reply:
> Remove the variables from breeze
Yep

> Set them in _initialization.sh (in the new function that you suggested)
Yep

And would be nice to remove other redundant ones, but it might be done separately. I might want to do some cleanup there :)

Reply:
Very nice. You've done exactly what I asked for :) . I just wanted to start explaining context so that you start understanding more broader context for it :)